### ISSUE TITLE
Course synopsis & target audience needed

### ISSUE THREAD

**Issue by tobyhodges** [tobyhodges/data-carpentry-embl-2018#1]
Hi @tavareshugo 

Our course & conference office has requested basic information to post a course webpage for the DC workshop. If you have time in the next few (working) days, could we pull together a short course summary, schedule (for the first two, "core", days at least), and description of the target student, please?

I'll create files and try to put first drafts together today.

**Comment by tavareshugo** [comment]
Hi @tobyhodges 

Sorry, I should have mentioned this before, but the course page should be set according to these instructions:
https://github.com/datacarpentry/workshop-template

I've done it, here: https://github.com/tavareshugo/2018-05-02-EMBL

I will adjust the course page as necessary, but feel free to raise issues there if some of the details are not correct or you get extra information (like instructors, helpers, room location, etc...).

Basically, that repo is just for the course page itself. We then link to the materials which live in other repos. 

I suppose we can still use this current repository for the actual materials of day 3, which are extra to the carpentry materials and then link to them from the course page?

**Comment by tavareshugo** [comment]
@tobyhodges 

I've only just noticed your synopsis document. I'll add you as a collaborator to the repo of the course page. Feel free to edit the `index.html` as you see fit. 

I'll meanwhile add your course description to it.
